firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -25,7 +25,8 @@ unsigned long state1_max_time = STATE1_MAX_TIME;
 unsigned long state2_max_duration = STATE2_MAX_DURATION;
 unsigned long state3_max_stillness_time = STATE3_MAX_STILLNESS_TIME;
 unsigned long state3_max_long_stillness_time =
-    STATE3_MAX_STILLNESS_TIME;  // By default, we use the same value for max_stillness_time and max_long_stillness_time
+    STATE3_MAX_STILLNESS_TIME;              // By default, we use the same value for max_stillness_time and max_long_stillness_time
+unsigned long state3_number_of_alerts = 0;  // The number of alerts generated in a session should start at 0
```

Comment:
Looks like it still has weird syntax highlighting. It might have something to do with the proceeding line having a comment. As the firmware linting and tests are passing, I'm going to ignore this, but good to note for the future.